stylesheets/mapLocations.css

```diff
@@ -0,0 +1,245 @@
+/* This file captures styling information for placement and overlay of the map. */
```

Comment:
I'd propose we name this file based on the entity type, for easier updates later. So just "maps.css". We could break it up and group later if needed.

Reply:
Ok